### ISSUE TITLE
Inconsistent formatting of raid vs egg

### ISSUE THREAD

**Issue by Mygod** [versx/RDMopole2#21]
Egg start/end time is timestamp but raid end time is in minutes.

**Comment by versx** [comment]
> Egg start/end time is timestamp but raid end time is in minutes.

This is intentional

**Comment by Mygod** [comment]
> > Egg start/end time is timestamp but raid end time is in minutes.
> 
> This is intentional

Explain yourself.

**Comment by versx** [comment]
![image](https://user-images.githubusercontent.com/1327440/93645409-cc977c80-f9b8-11ea-97d2-94d3770b141f.png)
Raid is ending in 10 minutes, no one cares when it started originally so it's replaced with a double hyphen `--`  

![image](https://user-images.githubusercontent.com/1327440/93645478-f6e93a00-f9b8-11ea-8765-004c0028dcc7.png)
Raid is still an egg so posting as the actual time was decided since people usually want to know the time that it starts and ends, maybe we could change start to i.e. 45m instead I guess. No one has ever mentioned anything since the original when it was decided.

**Comment by Mygod** [comment]
I don't think this table should display real time information. I think absolute timestamp makes more sense (and also maybe add raid duration?).

**Comment by Mygod** [comment]
For example your 10m does not become 9m after a minute.

**Comment by versx** [comment]
> For example your 10m does not become 9m after a minute.

Of course it does, why wouldn't it?

**Comment by Mygod** [comment]
Questionable design but ok.
